src/rpc/handlers/BookChanges.cpp

```diff
@@ -0,0 +1,250 @@
+#include <ripple/app/ledger/Ledger.h>
+#include <ripple/basics/ToString.h>
+
+#include <backend/BackendInterface.h>
+#include <rpc/RPCHelpers.h>
+
+#include <boost/json.hpp>
+#include <algorithm>
+
+namespace json = boost::json;
+using namespace ripple;
+
+namespace RPC {
+
+struct BookChange
+{
+    STAmount sideAVolume;
+    STAmount sideBVolume;
+    STAmount highRate;
+    STAmount lowRate;
+    STAmount openRate;
+    STAmount closeRate;
+};
+
+class BookChangesHandler
+{
+    std::reference_wrapper<Context const> context_;
```

Comment:
If you've deleted copy assignment, you can probably just use `Context const&`

Reply:
I'd rather not have const/ref members ever, no matter what

Reply:
Why do we want to avoid ref members?

Reply:
@cjcobb23 for me personally it's enough that [it's a core guideline](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#c12-dont-make-data-members-const-or-references). I understand it's ok if you know what you are doing but i'd try to avoid it most of the time. using `reference_wrapper` is totally fine - it's just a pointer inside and basically a `zero-cost abstraction`.